packages/pricing/src/types/repositories/currency.ts

```diff
@@ -0,0 +1,13 @@
+export interface CreateCurrencyDTO {
```

Comment:
question: Are all the new types exact duplicates, or do they need to be reviewed?

Reply:
basically duplicates, some has fewer props, such as the createPriceSetDTO for the service which was previously defined as a dependent on the api type: `Omit<createPriceListDTO, "rules" | "prices">`. 

No API changes are introduced, it's just separation in order to decouple the external and internal api's